docs/roadmap.md

```diff
@@ -121,5 +121,5 @@ You can download WinUI 3 Preview 1 to try out - we'd love feedback on it, althou
 
 See the image below for a more detailed look at our plans for getting WinUI 3.0 to market: 
 
-![Winui roadmap](images/winui-roadmap-img.PNG)
+![Winui roadmap](WinUI-RoadmapJuly2020.png)
```

Comment:
>RoadmapJuly2020 [](start = 23, length = 15)

should this be in the images directory?

Reply:
Whoops, yes, moving it in upcoming commit